### DISCUSSION TITLE
Easy Banners - category condition

### DISCUSSION THREAD

**Discussion by kerlama** [breezefront/community#23] [2. Ideas]
Hi,

I would like to point out that "Category" condition in Easy Banners module is missing "IS". It would be good if you can add **IS** condition, as if we use IS ONE OF, then banner is shown on both category page and product , which belongs to that category.

https://i.imgur.com/m5tIF57.png

/IS

**Comment by vovayatsyuk** [comment]
Try to combine this condition with the "Page" condition: Category is one of "11" and Page is one of "catalog_category_view".

p.s. Please do not use these discussions for non-breeze questions. Thanks!
